### PR TITLE
Merge a bunch of recipes

### PR DESCRIPTION
**Pull request by AndersonTorres**
Recipes added:

- adjust-parens
- beginend
- frame-mode
- nlinum
- pocket-reader
- sed-mode
- sql-indent
- stream
- svg
- vigenere

- auto-complete-yasnippet
- common-lisp-snippets

### CODE REVIEW

File: recipes/beginend.rcp [hunk 1]
```diff
@@ -0,0 +1,4 @@
+(:name beginend
+       :type github
+       :description "Redefine beginning-of-buffer and end-of-buffer to more meaningful locations"
+       :pkgname "/DamienCassou/beginend")
```
Comment (npostavs) [inline recipes/beginend.rcp]:
```suggestion
       :pkgname "DamienCassou/beginend")
```

File: recipes/frame-mode.rcp [hunk 2]
```diff
@@ -0,0 +1,6 @@
+(:name frame-mode
+       :description "Use frames instead of windows whenever display-buffer is called"
+       :type github
+       :pkgname "IvanMalison/frame-mode"
+       :url "https://github.com/IvanMalison/frame-mode"
```
Comment (npostavs) [inline recipes/frame-mode.rcp]: `:url` isn't needed for `:type github`

**Comment by AndersonTorres** [comment]
Well, then I'm a bit confused.

There is some reason to maintain the other `type: elpa` recipes in the repo? Or they are just "legacy"?

Anyway, I will update now.

**Comment by AndersonTorres** [comment]
@npostavs, now everything is OK!

File: recipes/pocket-reader.rcp [hunk 3]
```diff
@@ -0,0 +1,5 @@
+(:name pocket-reader
+       :description "Emacs client for Pocket reading list (getpocket.com)"
+       :type github
+       :pkgname "alphapapa/pocket-reader.el"
+       :url "https://github.com/alphapapa/pocket-reader.el")
```
Comment (npostavs) [inline recipes/pocket-reader.rcp]: `:url` isn't needed for `:type github`

Comment (AndersonTorres) [reply recipes/pocket-reader.rcp]: Okay!

**Pull request merged**